front-end/index.md

```diff
@@ -3,48 +3,51 @@ layout: page
 title: Front End Resources
 ---
 
-All assignments for Mod 0 are listed below and organized into segments based on the due date. Each segment of work is due prior to the next live session.
+All lessons and assignments for Mod 0 are listed below and organized into segments based on the due date. Reach out if you have any questions! 
 
-## Before Session 1
+## Pre-Work | Complete Before Session 1
 * [Complete all Computer Setup]({{ site.url }}/computer-setup) 
 * [Read the Syllabus]({{ site.url }}/syllabus) 
 * [Review the Mod 0 Calendar]({{ site.url }}/calendar)
-* [Prep Your "Why"]({{ site.url }}/shared/why)
-* [Asking Technical Questions]({{ site.url }}/shared/asking_qs) 
-* [Markdown]({{ site.url }}/shared/markdown)
-* [Introduction Video]({{ site.url }}/shared/intro-video)
-
-## Segment 1 - Completed Before Session 2
+* [Slack 101]({{ site.url }}/prework/slack101)
+* [Markdown]({{ site.url }}/prework/markdown)
+* [Terminal]({{ site.url }}/prework/terminal)
+* [VS Code]({{ site.url }}/prework/vs-code)
+* [Prep Your "Why"]({{ site.url }}/prework/why)
+* [Introduction Video]({{ site.url }}/prework/intro-video)
+
+## Segment 1
+### Live Sessions
 * [Calendaring & Time Management]({{ site.url }}/shared/calendaring)
-* [Terminal]({{ site.url }}/shared/terminal)
-* [VS Code]({{ site.url }}/shared/vs-code)
 * [Git Workflow]({{ site.url }}/shared/git)
-
-## Segment 2 - Completed Before Session 3
-* [GitHub]({{ site.url }}/shared/github)
 * [Data Types]({{ site.url }}/front-end/data_types)
-* [Pair Programming]({{ site.url }}/shared/pairing)
-* [Pairing Session 1]({{ site.url }}/shared/pairing_1)
-* <a href="https://airtable.com/appl0fblUIvOotJko/shrAI3LMVuM3cDwq6" target="_blank">Career Development Survey</a>
 
-## Segment 3 - Completed Before Session 4
-* [Conditionals]({{ site.url }}/front-end/conditionals)
+### Assignment - Complete Before Session 2
+* [Asking Technical Questions]({{ site.url }}/shared/asking_qs) 
+
+## Segment 2
+### Live Sessions
+* [GitHub]({{ site.url }}/shared/github)
 * [Arrays]({{ site.url }}/front-end/arrays)
-* [For Statements]({{ site.url }}/front-end/for)
+
+### Assignment - Complete Before Session 3
```

Comment:
Same here. I like the "homework" happening in the same segment, after the live lesson, so how it's structured here on the FE page.